### DISCUSSION TITLE
[Feature Request] Move to album based on "X"

### DISCUSSION THREAD

**Discussion by MinnesotaMean** [immich-app/immich#14605] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am a manufacturing engineer by trade and have a dream feature that would make my life incredibly more streamlined.
It would be incredible to have albums auto populate based on city location and/or time. For example if i take a picture in Minneapolis,MN between 5am and 5pm put it in this album and hide from main timeline. 

My wife and I share an Immich account login so both our phones photos appear on the same timeline -an AWESOME feature, this would be an addon to that, and I'm sure many other will find value in!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1673
